test/base/src/androidTest/kotlin/io/realm/test/shared/MutableRealmTests.kt

```diff
@@ -333,6 +333,25 @@ class MutableRealmTests {
         assertEquals("UPDATED", child.stringField)
     }
 
+    @Test
+    fun copytToRealm_existingListIsFlushed() {
```

Comment:
Should we also add a smoke test for lists of primitives? Judging from the code it should also work, but just be sure, also in the future?